base_layer/core/src/transactions/transaction/transaction_kernel.rs

```diff
@@ -126,13 +126,13 @@ impl Hashable for TransactionKernel {
     /// $$ H(feature_bits | fee | lock_height | P_excess | R_sum | s_sum)
     fn hash(&self) -> Vec<u8> {
```

Comment:
Should this not use consensus encoding as well?
This hash is committed to in the kernel mmr, and every single implementation should be able to calculate this.